### ISSUE TITLE
ISSUE when running Lammps  from Python

### ISSUE THREAD

**Issue by ZhanPengOu** [deepmodeling/deepmd-kit#1135]
Hi,developer

After trainning a model, i‘m going to run lammps from a Python script using  `lmp`, but  something error happened , lammps error messages are following:

`WARNING: lammps_gather_atoms: unsupported data type (src/library.cpp:2047)`

This WARNING caused by `lammps.gather_atoms("x",1,3)`.Same bug reported on the   [https://github.com/lammps/lammps/pull/2472](url)

It seem that there is a bug in version 29Oct2020, but fixed in latest version.
Anyway,what should i do to fix this problem if i don't want to download the latest deepmd-kit?
Looking forward to your reply
thanks

Deepmd version: v2.0.0b3

**Comment by njzjz** [comment]
lammps/lammps#2472 only modifies a Python file, so I think you can easily apply the patch. Using the following script to get the path:
```py
import lammps
print(lammps.__file__)
```

**Comment by ZhanPengOu** [comment]
for the issue#lammps/lammps#2472 , #lammps/lammps#2466 change the file `lammps/src/library.cpp`, but i can't found this file in deepmd-kit folder

**Comment by njzjz** [comment]
A new version of lammps-dp package has been released here https://conda.deepmodeling.org/linux-64/lammps-dp-2.0.1-h9954876_1.tar.bz2 and you can use `conda` to install it.

**Comment by ZhanPengOu** [comment]
thanks for your help!
problem solved
